### PR TITLE
Updated Sqlite3

### PR DESCRIPTION
**Pull request by mmottl**

### CODE REVIEW

File: packages/sqlite3/sqlite3.4.1.1/opam [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  [["debian"] ["libsqlite3-dev"]]
+  [["freebsd"] ["database/sqlite3"]]
+  [["openbsd"] ["database/sqlite3"]]
+  [["ubuntu"] ["libsqlite3-dev"]]
+  [["centos"] ["sqlite-devel"]]
+  [["rhel"] ["sqlite-devel"]]
+  [["fedora"] ["sqlite-devel"]]
+  [["alpine"] ["sqlite-dev"]]
+  [["osx" "homebrew"] ["sqlite3"]]
+  [["osx" "macports"] ["sqlite3"]]
```
Comment (avsm) [inline packages/sqlite3/sqlite3.4.1.1/opam]: any chance you could add `[["opensuse" ["sqlite3-devel"]]` to the depexts list? The CI now tests that too and it'll increase your support coverage. Not a blocker to merge though.

**Comment by avsm** [comment]
why close this?

File: packages/sqlite3/sqlite3.4.1.1/opam (hunk 1 above)
Comment (mmottl) [inline packages/sqlite3/sqlite3.4.1.1/opam]: @avsm Thanks for the hint, will do.  Had to close this due to a serious issue with the release.  I removed it before anybody could download it.  I had to update my release scripts.  I'll submit a new pull request soon.

Comment (mmottl) [inline packages/sqlite3/sqlite3.4.1.1/opam]: Actually, the problem was with the release including the `opensuse` patch, which, of course, has a new release number.  4.1.1 was ok.

**Comment by mmottl** [comment]
Ok, I have cleaned up the mess.  I just skipped 4.1.1 in OPAM.

**Comment by mmottl** [comment]
I hope my changes haven't broken this pull request.  Some new checks have failed, apparently because they were trying to do something with the non-existing SQLite3 4.1.1 package.  Anything I can or should do about that?  There is no reference to 4.1.1 anymore in the fork I'm trying to merge.  Not sure whether packages in intermediate patches need to remain in existence forever.

**Comment by mmottl** [comment]
I'll try to resubmit this pull request, maybe it's going to work this time.

**Pull request closed without merging**